community-containers/nextcloud-exporter/readme.md

```diff
@@ -0,0 +1,43 @@
+## Prometheus Nextcloud Exporter
+
+A Prometheus exporter that collects metrics from your Nextcloud instance for monitoring and alerting.
+
+### What it does
+- Monitors user activity, file counts, and storage usage
+- Tracks system health, database size, and app statistics  
+- Exposes metrics on port `9205` for Prometheus to scrape
+- Uses Nextcloud's serverinfo API with secure token authentication
+
+### Use cases
+- Monitor Nextcloud performance and capacity
+- Set up alerts for system issues or resource limits
+- Create dashboards showing usage trends and system health
+- Track active users and storage growth over time
+
+### Prometheus Configuration
+Add this to your `prometheus.yml` to scrape the exporter:
+
+```yaml
+scrape_configs:
+  - job_name: 'nextcloud'
+    scrape_interval: 90s
+    static_configs:
+      - targets: ['your-nextcloud-domain.com:9205']
+    metrics_path: /metrics
+    scheme: http
+```
+
+Replace `your-nextcloud-domain.com` with your actual Nextcloud domain.
+
+### Notes
+Automatically configures the required serverinfo token for authentication between Nextcloud and the exporter. 
+
+Compatible with Grafana for visualization (dashboard ID `20716` available).
+
+https://grafana.com/grafana/dashboards/20716-nextcloud/
+
```

Comment:
Please also add a note how to add it to the aio stack, for example `See https://github.com/nextcloud/all-in-one/tree/main/community-containers#community-containers how to add it to the AIO stack`